rethinkdb/datadog_checks/rethinkdb/queries.py

```diff
@@ -0,0 +1,148 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from . import operations
+from .document_db import DocumentQuery
+
+# System configuration.
+
+# See: https://rethinkdb.com/docs/system-tables/#configuration-tables
+config_summary = DocumentQuery(
+    source=operations.get_config_summary,
+    name='config_summary',
+    prefix='rethinkdb.config',
+    metrics=[{'type': 'gauge', 'path': 'servers'}, {'type': 'gauge', 'path': 'databases'}],
```

Comment:
Types could be constants: `gauge`, `monotonic_count`, etc.

I know any error will be caught by type checking, but still think that using constants is appropriate here.

I think it has been already discussed, so, if you don't agree, it's fine, you can ignore this comment. :)

Reply:
I think such constants should be provided at the base package level — seems like we're duplicating that info in a lot of integrations already.

So I suggest to keep it like this, and then retrofit if/when we introduce constants via `datadog_checks_base`…?